openedx_webhooks/ui.py

```diff
@@ -28,7 +28,7 @@ def index():
 
 
     jira_username = None
-    if jira_session.authorized:
+    if jira_session.authorized and settings.JIRA_SERVER:
```

Comment:
This check looks backwards, but I guess `.authorized` here might mean, `connected anonymously to jira`?

Reply:
Yea, I'm not sure, I can reverse them but right now `jira_session.authorized` is returning `True` even when there are no jira settings set  and I'm not really sure how deep to dig into flask-dance to figure out what's going on.